ftrip.io.catalog-service/ftrip.io.catalog-service/Accommodations/UseCases/CreateAccommodation/CreateAccommodationRequestHandler.cs

```diff
@@ -23,6 +24,7 @@ public class CreateAccommodationRequestHandler : IRequestHandler<CreateAccommoda
         private readonly IMapper _mapper;
         private readonly IStringManager _stringManager;
         private readonly IMessagePublisher _messagePublisher;
+        private readonly ILogger _logger;
 
         public CreateAccommodationRequestHandler(
```

Comment:
I guess we can increase num of parameters. Here, we are injecting 4 utility classes so I feel like this handler is still fine.